java.header

```diff
@@ -1,15 +1,29 @@
 ^/\*$
-^ \* Copyright \d\d\d\d,? Google (Inc\.|LLC)( All [rR]ights [rR]eserved\.)?$
+^ \* Copyright 20\d\d,? Google (Inc\.|LLC)( All [rR]ights [rR]eserved\.)?$
 ^ \*$
-^ \* Licensed under the Apache License, Version 2\.0 \(the "License"\);$
-^ \* you may not use this file except in compliance with the License\.$
-^ \* You may obtain a copy of the License at$
+^ \* Redistribution and use in source and binary forms, with or without$
+^ \* modification, are permitted provided that the following conditions are$
+^ \* met:$
 ^ \*$
-^ \*[ ]+https?://www.apache.org/licenses/LICENSE-2\.0$
+^ \*     \* Redistributions of source code must retain the above copyright$
+^ \* notice, this list of conditions and the following disclaimer.$
+^ \*     \* Redistributions in binary form must reproduce the above$
+^ \* copyright notice, this list of conditions and the following disclaimer$
+^ \* in the documentation and/or other materials provided with the$
+^ \* distribution.$
+^ \*     \* Neither the name of Google (Inc\.|LLC) nor the names of its$
+^ \* contributors may be used to endorse or promote products derived from$
+^ \* this software without specific prior written permission.$
 ^ \*$
-^ \* Unless required by applicable law or agreed to in writing, software$
-^ \* distributed under the License is distributed on an "AS IS" BASIS,$
-^ \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.$
-^ \* See the License for the specific language governing permissions and$
-^ \* limitations under the License\.$
-^ \*/$
+^ \* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS$
+^ \* "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT$
+^ \* LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR$
+^ \* A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT$
+^ \* OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,$
+^ \* SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES \(INCLUDING, BUT NOT$
+^ \* LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,$
+^ \* DATA, OR PROFITS; OR BUSINESS INTERRUPTION\) HOWEVER CAUSED AND ON ANY$
+^ \* THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT$
+^ \* \(INCLUDING NEGLIGENCE OR OTHERWISE\) ARISING IN ANY WAY OUT OF THE USE$
+^ \* OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.$
+^ \*/$
```

Comment:
This is the same as `license-header-javadoc.txt` which gradle has been using. Nothing about the license changes except that maven uses this file. 

I've updated it to accept Google Inc and LLC